src/pages/Pomodoro/lib/SettingsContext.tsx

```diff
@@ -0,0 +1,101 @@
+import {createContext, useState} from "react";
+// import style from "../Logic/Settings.module.css";
+
+export const SettingContext = createContext({});
+
+const SettingsContextProvider = (props) => {
+
+    const [pomodoro, setPomodoro] = useState(0);
+    // this if from the settings page
+    const [executing, setExecuting] = useState({});
+    const [startAnimate, setStartAnimate] = useState(false);
+
+    function startTimer(){
+        setStartAnimate(true);
+    }
+
+    function pauseTimer(){
+        setStartAnimate(false);
+    }
+
+    function stopTimer(){
+        setStartAnimate(false);
+    }
+
+    function restartTimer() {
+
+        //NOT WORKING
```

Comment:
are you still working on this PR to fix the restartTimer function or can I review it / approve it?